src/components/article/ArticleFullCard.jsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+import { getArticleById } from "../../utils/api";
+import { getDateString } from "../../utils/date";
+import Loading from "../Loading";
+
+function ArticleFullCard({ article_id, article, setArticle }) {
+	const [isLoading, setIsLoading] = useState(false);
+
+	useEffect(() => {
+		setIsLoading(true);
+		getArticleById(article_id).then(data => {
+			setArticle(data.article);
+			setIsLoading(false);
+		});
+	}, [article_id, setArticle]);
```

Comment:
Dont need setArticle as a dependency as its a function, not a variable; it wont be changing